webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/GroupService.scala

```diff
@@ -36,6 +38,12 @@ final case class GroupService(
       status = knoraGroup.status.value,
       selfjoin = knoraGroup.hasSelfJoinEnabled.value,
     )
+
+  def createGroup(request: GroupCreateRequest, project: Project): Task[Group] =
+    for {
+      knoraGroup <- knoraGroupService.createGroup(request, project)
+      group      <- toGroup(knoraGroup)
+    } yield group
```

Comment:
*Non-blocking:* this might be more character-efficient with `.createGroup(..).flatMap(toGroup)`.